tests/robot-tests/tests/admin_and_public/bau/footnotes.robot

```diff
@@ -50,6 +50,18 @@ Upload subject
     user checks headed table body row contains  Data file size   17 Kb  ${section}
     user checks headed table body row contains  Status           Complete  ${section}  360
 
+Add meta guidance to subject
+    [Tags]  HappyPath
+    user clicks link  Metadata guidance
+    user waits until h2 is visible  Public metadata guidance document
+
+    user waits until page contains accordion section  ${SUBJECT_NAME}
+    ${text_element}=  user gets meta guidance accordion text element  ${SUBJECT_NAME}
```

Comment:
Given the suggest keyword change, this would be something like `${editor}` instead